src/index.ts

```diff
@@ -36,6 +37,26 @@ export async function main() {
     )
     core.endGroup()
 
+    core.startGroup("Load json files into context")
+    const files = core.getInput("json-files-to-include")
+    if (files !== "") {
+      const filesJson = JSON.parse(files)
+      // loop through and load each file into context with the name and files json contents
+      for (const file of filesJson) {
+        const name = file.name
+        const filepath = file.path
```

Comment:
```suggestion
```

Nit: I would just use the dot notation here, we only have two usages of `file.path` and one usage of `file.name`, it's better to just access via the parent object here for readability via the additional context it gives.